src/ontology/mondo.Makefile

```diff
@@ -409,6 +409,22 @@ update-rare-subset:
 ##### Mondo External Content Pipeline ################
 ######################################################
 
+####################################
+##### OMIM #####################
+####################################
+
+.PHONY: update-omim-genes
+update-omim-genes:
+	$(MAKE) $(TMPDIR)/external/processed-mondo-omim-genes.robot.owl -B
+	# We need to be less aggressive here, as some gene relations were not originally sourced
+	# from OMIM, and were added, for example, for ClinGen.
+	grep -vE '^(relationship: has_material_basis_in_germline_mutation_in .*source="OMIM:)' $(SRC) > $(TMPDIR)/mondo-edit.tmp || true
```

Comment:
Currently, only OMIM is used as the source for `has_material_basis_in_germline_mutation_in`. However, this update should only remove the axioms where OMIM is the source in case there are different sources in the future. 
cc: @matentzn

Reply:
Just note that at least once we should clear out all these axioms, to not ignore some that right now dont have omim as the source?

Reply:
my comment makes zero sense given:

> only OMIM is used as the source for has_material_basis_in_germline_mutation_in

it would make sense if:

1. there are has_material_basis_in_germline_mutation_in axioms which have no evidence
2. or evidence other than omim

as I think your statement would then not delete them; and it should!

Reply:
deleted, but not committed yet